### ISSUE TITLE
[1.12.2] Some Glow-in-the-Dark Textures appear to be... fading out, in a way if blocks surround them. Kinda hard to explain without showing an image.

### ISSUE THREAD

**Issue by SonicX8000** [Chisel-Team/ConnectedTexturesMod#62]
Minecraft: 1.12.2
Forge: 14.23.4.2708

Mods:
CTM-MC1.12-0.3.0.15
twilightforest-1.12.2-3.7.424-universal

---
Now... I am not sure if it's meant to be here but I noticed this after updating CTM so I figure I mention it here first.

Anyway... using the Twilight Forest Mod since... as far as I know... has these blocks that have some parts of their textures that glow in the dark without any light level... however apparently with the latest CTM mod it kinda broke some of those glowing bits of the texture if blocks are surrounding them. It's perhaps easier to notice it in some screenshots...

***Image Key***
Left is "CTM-MC1.12-0.2.3.12", Right is "CTM-MC1.12-0.3.0.15"
Blocks from Mod: "twilightforest-1.12.2-3.7.424-universal"

Example Image 1:
![](https://cdn.discordapp.com/attachments/377171551155650560/458699217196810242/twilightforestglowCTMBlocks.PNG)

In this image... the blocks have this glow-in-the-dark effect which are the Castle Rune Brick, Reappearing Block, Castle Door Blocks, to name a few.

If you look at the Right Side of the Image, you can see how they glow and such, but when you look at the Left Side, the Castle Door Blocks appear to be slightly faded while some of the other block don't seem to be affected. While it's not noticeable in this image it becomes more clear in this next image.

Example Image 2:
![](https://cdn.discordapp.com/attachments/377171551155650560/458701514173841438/twilightforestglowCTMCastleDoor.PNG)

This one shows it more clearly when you surround the Castle Door Blocks. Right Side of the Image shows the glowing effect kinda being affected but that may be normal due to the way the lighting works however when you look at the Left Side of the Image, it's glow effect is pretty much almost gone.

It could be possible that maybe the Mods that use this effect might need to update due to the changes with CTM but I figure I mention this just to see if it's CTM or something. Unless there's a config in CTM that does this?

**Comment by tterrag1098** [comment]
Could anyone experiencing this issue please test with this version? https://drive.google.com/file/d/1Kp4aabRJY8QGxoxyT9p2HoCkf0nCI3yj/view?usp=sharing

This version reverts this commit: https://github.com/Chisel-Team/ConnectedTexturesMod/commit/4b137cdeae5dc5077397a90f98ab6e522d00989e

**Comment by SonicX8000** [comment]
![](https://cdn.discordapp.com/attachments/377171551155650560/458758984439037983/twilightforestglowCTMDEVBlocksFixed.PNG)

Appears to be fixed in that file that you linked. Glow effect looks to be normal especially for the Castle Door Blocks.

**Comment by tterrag1098** [comment]
Fortunately the forge bug I was going to blame literally just got put into a build. Could you try the latest build of forge (2709) and see if it also fixes the problem?

**Comment by SonicX8000** [comment]
![](https://cdn.discordapp.com/attachments/377171551155650560/459102190288502786/twilightforestglowCTMForge2709BlocksFixedLavaLeak.PNG)
Using "Forge: 14.23.4.2709" with "CTM-MC1.12-0.3.0.15"... the glowing effect doesn't appear to be faded out anymore... however I noticed something as I was taking the picture.

In the red circles, it looks like the lava texture is trying to bleed through the Castle Door Block. Is that suppose to happen? Looking back at the images that I posted here, it happens in "CTM-MC1.12-0.2.3.12" as well. There's no Lava behind the blocks and when you move your viewpoint the lines are more noticeable.
